phpunit.xml.dist

```diff
@@ -8,6 +8,9 @@
   <filter>
     <whitelist>
       <directory suffix=".php">src</directory>
+      <exclude>
+        <directory suffix=".php">src/Logging/V2</directory>
```

Comment:
What would everyone think of using globs here to cover these without needing to add more excludes for every new API surface?

`src/*/V*`, or `src/*/V1`, `src/*/V2` should cover it.